_posts/en/pages/2016-01-15-lifecycle.md

```diff
@@ -57,12 +57,16 @@ _TBA: to be announced_
 
 The description above only describes Bitcoin Core software releases. Many other parts of the Bitcoin system contain their own versions.  A few examples:
 
-- Every **block** and **transaction** contains a version number.
+- Every **transaction** contains a version number.
+- Every **block** allows miners to signal their readiness to enforce new
+  versions of the consensus rules.
```

Comment:
I think we should just remove blocks here, since it's no longer part of the example.